src/tasks/containers/TaskPage.tsx

```diff
@@ -144,6 +147,11 @@ class TaskPage extends PureComponent<Props> {
     // currently we delete that part of the script
     script = script.replace(new RegExp('option\\s+task\\s+=\\s+{(.|\\s)*}'), '')
 
+    if (!isValidFlux(script)) {
+      this.props.invalidFlux()
+      return
```

Comment:
Doing the check here in the TasksPage, allows us to keep the "Create Task" open on failure. Therefore, the user doesn't "lose their work" as they try creating a new task.